### ISSUE TITLE
Scale step prefix & suffix are not read out by the screen reader

### ISSUE THREAD

**Issue by joe-allen-89** [adaptlearning/adapt-contrib-slider#211]
### Subject of the issue/enhancement/features
The scaleStepPrefix & scaleStepSuffix are currently not read out by the screenreader, 

### Your environment

- FW v5.40.3
- Latest Firefox
- Windows, NVDA

### Steps to reproduce
Create a course with a slider, set the scaleStepPrefix/scaleStepSuffix and navigate through the course using a screenreader/keyboard

### Expected behaviour
The scaleStepPrefix/scaleStepSuffix should be read out to ensure screenreader users don't miss any information

### Actual behaviour
They are not read out
